docs/reference/api/containers/splitcontainer.rst

```diff
@@ -20,30 +22,49 @@ Usage
 
     import toga
 
-    split = toga.SplitContainer()
     left_container = toga.Box()
     right_container = toga.ScrollContainer()
 
-    split.content = [left_container, right_container]
+    split = toga.SplitContainer(content=[left_container, right_container])
 
-Setting split direction
------------------------
-
-Split direction is set on instantiation using the ``direction`` keyword argument. Direction is vertical by default.
+Content can be specified when creating the widget, or after creation by assigning
+the ``content`` attribute. The direction of the split can also be configured, either
+at time of creation, or by setting the ``direction`` attribute:
 
 .. code-block:: python
 
     import toga
 
     split = toga.SplitContainer(direction=toga.SplitContainer.HORIZONTAL)
+
     left_container = toga.Box()
     right_container = toga.ScrollContainer()
 
     split.content = [left_container, right_container]
 
+By default, the content of the SplitContainer will be evenly divided between the
+content. To specify an uneven split, you can provide a flex value when specifying
+content. In the following example, there will be a 60/40 split between the left
+and right panels.
+
+.. code-block:: python
+
+    import toga
+
+    split = toga.SplitContainer()
+    left_container = toga.Box()
+    right_container = toga.ScrollContainer()
+
+    split.content = [(left_container, 3), (right_container, 2)]
+
+This only specifies the initial split; the split can be modified by the user
+once it is displayed. If a flex weight isn't specified, a value of 1 is assumed.
+The minimum flex weight is 1; any smaller value will be interpreted as 1.
```

Comment:
This was a safety mechanism; specifying "flex=0" is a an easy trap to fall into, and any negative value will also be a problem. I guess it would be safer to just reject "flex<=0" as an error, and let users specify weights as "0.6/0.4". I'll update to reflect this.